srcpkgs/libopensmtpd/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'libopensmtpd'
+pkgname=libopensmtpd
+version=0.7
+revision=1
+build_style=gnu-makefile
+make_cmd="make -f Makefile.gnu"
+make_use_env=yes
+make_build_args="MANFORMAT="
+make_install_args="MANFORMAT="
+makedepends="libevent-devel"
+short_desc="Library for creating opensmtpd filters"
+maintainer="Adam Gausmann <agausmann@fastmail.com>"
+license="ISC"
+homepage="http://imperialat.at/dev/libopensmtpd/"
+distfiles="https://distfiles.sigtrap.nl/${pkgname}-${version}.tar.gz"
+checksum=cdd6d5e11a7063f9265da43a42309c646554b8b68b03982ca8472dbe2b08a613
+
+do_check() {
+    : # `make check` fails: there is no check/test target
+}
```

Comment:
that's usually just a warning which doesn't mean we have to explicitly disable tests because do_check passes

Reply:
It causes the CI to fail; adding this fixed it.

Reply:
See previous rev 3efbeb96077f578f8a257b9b4e3cd7f572d38029

Reply:
I see, it's not ignored when you set make_cmd. Could you use `make_check=no # <description>` instead?

Reply:
Thank you!